Sources/Paywalls/Components/PaywallComponentPropertyTypes.swift

```diff
@@ -10,13 +10,15 @@
 //  PaywallComponentPropertyTypes.swift
 //
 //  Created by James Borthwick on 2024-08-29.
-
 // swiftlint:disable missing_docs
 
 import Foundation
 
 #if PAYWALL_COMPONENTS
 
+public typealias TierId = String
+public typealias ColorHex = String
```

Comment:
I see these were added as an extension to `PaywallComponent` above, so maybe we can remove these?

Reply:
Good catch! I also deleted `TierId` entirely because it's not used at this point.